orunmila.go

```diff
@@ -268,16 +268,11 @@ func createDbFileifNotExists(dbPtr string) {
 	}
 }
 
-// parse args of the import subcommand and exec it
-func vacuumSubcmd(args []string) {
-	flag := flag.NewFlagSet("vacuum", flag.ContinueOnError)
-
-	flag.Parse(args)
-
-	createDbFileifNotExists(*dbPtr)
+// parse args of the vaccum subcommand and exec it
+func vacuumSubcmd() {
+	// parse the global args instead (debug && db path)
 
 	dsn := fmt.Sprintf("file:%s?mode=rw", *dbPtr)
-	createDB(dsn)
 	db, err := sql.Open("sqlite3", dsn)
 	check(err)
 
```

Comment:
Also can you move the defer to be right after check(err) and add an empty line after it?